docs/components/DrawerDocs.js

```diff
@@ -78,7 +80,16 @@ const DrawerDocs = React.createClass({
         <p>This number represents the percent of the screen visible between the left edge of the screen and the left edge of the drawer.</p>
 
         <h5>breakPoints<label>Object</label></h5>
-        <p></p>
+        <p>This object takes 2 properties:</p>
+        <ul style={styles.unorderdLists}>
+          <li style={styles.listItem}><h5 style={styles.listItem}>large <label>Number</label></h5></li>
+          <li style={styles.listItem}><h5 style={styles.listItem}>medium <label>Number</label></h5></li>
+        </ul>
+        <p>When the screen size is: <br />
+          &#62;&#61; large: drawer width is maxWidth <br />
+          &#60;	large and > medium: drawer width is 20% from left side <br />
```

Comment:
Looks like you need to change `>` to `&#62;`. Other than that I think this is 👍  to 🚢 .